frontend/src/static/css/includes/_theme_variables.scss

```
@@ -22,9 +22,9 @@ $color-error-red: rgba(red, 0.8);
 
 $message-default-color: rgba(#111, 0.9);
 
-$message-default-bg-color: #e6e6e6;
-$message-info-bg-color: #e6e6fa;
-$message-error-bg-color: #fae6e6;
-$message-success-bg-color: #e6f0e6;
+$message-default-bg-color: var(--cinemata-neutral-200);
```

Stylelint (17.11.1)

[error] Unexpected empty line before $-variable (scss/dollar-variable-empty-line-before)

(scss/dollar-variable-empty-line-before)